src/app/shared/components/template/components/layout/layout.ts

```diff
@@ -49,7 +49,9 @@ export class TemplateLayoutComponent implements ITemplateRowProps, OnInit {
   public scrollToTop() {
     let parent = this.parent;
     while (parent) {
-      parent.elRef.nativeElement.scrollTop = 0;
+      if (parent.elRef) {
+        parent.elRef.nativeElement.scrollTop = 0;
+      }
       parent = parent.parent;
```

Comment:
When evaluating a template that is not rendered it will not have an element reference. This PR doesn't change that, but just adds a catch for code that was missing